packages/shared/src/hooks/useMyFeed.ts

```diff
@@ -25,6 +30,7 @@ export function useMyFeed(): UseMyFeed {
   const { user } = useContext(AuthContext);
   const { flags } = useContext(FeaturesContext);
   const shouldShowMyFeed = isFeaturedEnabled(Features.MyFeedOn, flags);
+  const myFeedPosition = getFeatureValue(Features.MyFeedPosition, flags);
```

Comment:
We want to use our new function that chooses a value out of a set of possibilities otherwise fall back to the default `sidebar`. We might introduce new variations in the future and don't want the version to break.

Reply:
This is the case, the defaults are managed in the `featureManagement` class.

See here: 

https://github.com/dailydotdev/apps/blob/1dee521b54563cb0e9224dc2a9fb7a2cad1561d2/packages/shared/src/lib/featureManagement.ts#L33

```js
  static readonly MyFeedPosition = new Features('my_feed_position', 'sidebar');
```

Reply:
It's not enough actually. Let's say in this version we support two values for this feature `sidebar` and `feed_ad` (I know that in reality there's more but just an example). Then the next version comes and we want to add a new type `header`. We set it in Flagsmith and now the old version breaks. Whereas if we check in every version of the received value is one of the supported values and if not fall back to default, it will be backward compatible.